docs/changes/coordinates/11449.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Removed multiple calls to ```transform_to``` from ```match_to_catalog_sky```
```

Comment:
I realize now that we are not really fixing any bug (contrary to what we thought), so really this is a performance improvement. Could you rename to `11449.other.rst`, replace "multiple" with "duplicate" and add the end add ", improving their performance."?